bin/rucio

```diff
@@ -1478,6 +1478,7 @@ def list_rules(args):
                   '%s[%d/%d/%d]' % (rule['state'], rule['locks_ok_cnt'], rule['locks_replicating_cnt'], rule['locks_stuck_cnt']),
                   rule['rse_expression'],
                   rule['copies'],
+                  sizefmt(rule['bytes']) if rule['bytes'] is not None else 'N/A',
```

Comment:
Forgive me, I forgot about something.

https://github.com/rucio/rucio/blob/a73004e1e6ecc5815c2c11a4c80e641be0b97930/bin/rucio#L1861

For completeness, you need to pass `args.human` as the second argument to `sizefmt()`. This also applies to the other call too.